Sources/Purchasing/Offering.swift

```diff
@@ -215,17 +215,37 @@ import Foundation
 
 extension Offering {
 
-    /**
-     - Returns: the `metadata` value associated to `key` for the expected type,
-     or `default` if not found, or it's not the expected type.
-     */
+    ///
+    /// - Returns: the `metadata` value associated to `key` for the expected type,
+    /// or `default` if not found, or it's not the expected type.
+    ///
     public func getMetadataValue<T>(for key: String, default: T) -> T {
         guard let rawValue = self.metadata[key], let value = rawValue as? T else {
             return `default`
         }
         return value
     }
 
+    ///
+    /// - Returns: the `metadata` value associated to `key` for the expected `Decodable` type,
+    /// or `nil` if not found.
+    /// - Throws: Error if the content couldn't be deserialized to the expected type.
+    ///
```

Comment:
Should we mention which `KeyDecodingStrategy` its using?

Reply:
Done.